src/azure-cli-core/azure/cli/core/cloud.py

```diff
@@ -59,49 +59,73 @@ class CloudSuffixNotSetException(CLIError):
 
 class CloudEndpoints:  # pylint: disable=too-few-public-methods,too-many-instance-attributes
 
+    ARM_INDEXES = {
```

Comment:
Singular form of "index" should suffice. Also, I suggest making it more explicit:

```suggestion
    ARM_METADATA_INDEX = {
```